the-alexa-skill/typescript/lib/the-alexa-skill-stack.ts

```diff
@@ -38,13 +38,10 @@ export class TheAlexaSkillStack extends cdk.Stack {
       removalPolicy: cdk.RemovalPolicy.DESTROY
     });
 
-    // Install Dependencies and Compile Lambda Function
-    execSync('cd lambda-fns && npm i && npm run build');
```

Comment:
On the other stacks, the build process is performed by "npm run build" and then the stack is deployed using the "cdk deploy" command. I think this should be kept, correct?